src/Core/Field.h

```diff
@@ -808,11 +808,29 @@ auto & Field::safeGet()
 template <typename T>
 T & Field::reinterpret()
 {
+    assert(which != Types::String); // See specialization for char
     using ValueType = std::decay_t<T>;
     ValueType * MAY_ALIAS ptr = reinterpret_cast<ValueType *>(&storage);
```

Comment:
Rewrite with `unalignedLoad`?

Reply:
`unalignedLoad` returns a new object and then the compiler might decide it doesn't need to create it, and `reinterpret` always returns a reference to an object built on top of the existing memory. I'm not sure if they are equivalent.